### DISCUSSION TITLE
Go-To-Next-Page-Auto links

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#46] [General]
Hi,

am I right that go-to-next-page-auto refers to the nearest artboard by x-position?

Is it possible to force these action to go to a linked artboard. F.e. when return to the „start-page“ in a row, this feature won't work because if a far x-position  (see image)

<img width="1121" alt="img" src="https://user-images.githubusercontent.com/2456068/127044349-610a1651-54b3-4627-9536-6d2647dd097f.png">

**Comment by MaxBazarov** [comment]
@Nevu 
Yes, go-to-next-page-auto refers to the next artboard. Sort order is defining in Configure Export dialog.

<img width="517" alt="Screenshot 2021-07-29 at 11 39 26" src="https://user-images.githubusercontent.com/18418306/127460510-564996d5-b29c-402b-a54a-af1dbc224a65.png">

Do I understand correctly you want to cycle transitions to switch from last page to the first automatically?

**Reply by Nevu** [reply]
Ok, didn't recognize these settings!

Yes, I want to cycle transitions in this case! But this doesn't work with these workflows I guess ...

**Reply by MaxBazarov** [reply]
I will fix it. If the last page also has auto transition enabled then it should transit to the first page. It sounds reasonable.

**Reply by MaxBazarov** [reply]
@Nevu 
Fixed in 16.6.3 
Example: https://github.com/ingrammicro/puzzle-publisher/raw/master/examples/PageTransition.sketch
